tests/Earthfile

```diff
@@ -300,6 +300,25 @@ secrets-test:
         --should_fail=true \
         --output_contains="unable to lookup secret SECRET3: not found"
 
+project-secrets-test:
```

Comment:
Where is this target being called from?

where is `project-secrets.earth`, did you forget a `git add`?

Reply:
Fixed. Thanks.

Reply:
I'm confused, I don't see a `BUILD +project-secrets-test` anywhere. Where did this get fixed?